.github/workflows/main.yml

```diff
@@ -245,11 +269,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v5
       - name: Install Rust Nightly
-        uses: actions-rust-lang/setup-rust-toolchain@v1
+        uses: actions-rust-lang/setup-rust-toolchain@v1.15.2
         with:
-          toolchain: nightly-2025-08-06-x86_64-unknown-linux-gnu
+          toolchain: nightly-2025-11-06-x86_64-unknown-linux-gnu
```

Comment:
This looks off already from the old version - we should be installing the latest version of nightly if we really want it to be a "nightly" build. This is just going to be "some random old version from nightly channel" which nobody will expect. Can we remove the extra suffix and just use "nightly"?